sherpa/astro/ui/utils.py

```diff
@@ -3789,8 +3789,7 @@ def get_backscal(self, id=None, bkg_id=None):
         References
         ----------
 
-        .. [1] "The OGIP Spectral File Format", Arnaud, K. & George, I.
-               http://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/spectra/ogip_92_007/ogip_92_007.html
+        `Arnaud., K. & George, I., "The OGIP Spectral File Format" <http://heasarc.gsfc.nasa.gov/docs/heasarc/ofwg/docs/spectra/ogip_92_007/ogip_92_007.html>`_
```

Comment:
Should the reference for this actually be:

Arnaud, K. A., George, I. M., & Tennant, A. F., "The OGIP Spectral File Format"? 

Note removal of period after Arnaud and addition of Allyn Tennant as co/secondary author. The three are listed on the heasarc link. (Middle initials can be added or omitted)